src/common/api.ts

```diff
@@ -291,3 +292,100 @@ export const requestClassList = async (
   };
   return handleAxios<ApiSpeciesData>(request);
 };
+
+export const batchUploadInit = async (
+  backendUrl: string,
+  uuid: string,
+  containerUuid: string,
+  nbPictures: number,
+): Promise<{
+  session_id: string;
+}> => {
+  if (backendUrl === "" || backendUrl == null) {
+    throw new ValueError("Backend URL is null or empty");
+  }
+  if (uuid === "" || uuid == null) {
+    throw new ValueError("UUID is null or empty");
+  }
+  if (containerUuid === "" || containerUuid == null) {
+    throw new ValueError("Container UUID is null or empty");
+  }
+  if (nbPictures === 0 || nbPictures == null) {
+    throw new ValueError("Number of pictures is null or empty");
+  }
+  const request = {
+    method: "post",
+    url: `${backendUrl}/new-batch-import`,
+    headers: {
+      "Content-Type": "application/json",
+      "Access-Control-Allow-Origin": "*",
+    },
+    data: {
+      user_id: uuid,
+      container_name: containerUuid,
+      nb_pictures: nbPictures,
+    },
+  };
+  return handleAxios<{
+    session_id: string;
+  }>(request);
+};
+
+export const batchUploadImage = async (
+  backendUrl: string,
+  data: BatchUploadMetadata,
+): Promise<boolean> => {
+  const {
+    containerName,
+    uuid,
+    seedId,
+    seedName, // TODO: remove when backend is implemented
+    zoom,
+    seedCount,
+    sessionId,
+    imageDataUrl,
+  } = data;
+  if (backendUrl === "" || backendUrl == null) {
+    throw new ValueError("Backend URL is null or empty");
+  }
+  if (sessionId === "" || sessionId == null) {
+    throw new ValueError("Session ID is null or empty");
+  }
+  if (imageDataUrl === "" || imageDataUrl == null) {
+    throw new ValueError("Image is null or empty");
+  }
+  if (containerName === "" || containerName == null) {
+    throw new ValueError("Container name is null or empty");
+  }
+  if (uuid === "" || uuid == null) {
+    throw new ValueError("UUID is null or empty");
+  }
+  if (seedId === "" || seedId == null) {
+    throw new ValueError("Seed ID is null or empty");
+  }
+  if (zoom === 0 || zoom == null) {
+    throw new ValueError("Zoom is null or empty");
+  }
+  if (seedCount === 0 || seedCount == null) {
+    throw new ValueError("Seed count is null or empty");
+  }
+  const request = {
+    method: "post",
+    url: `${backendUrl}/upload-picture`,
+    headers: {
+      "Content-Type": "application/json",
+      "Access-Control-Allow-Origin": "*",
+    },
+    data: {
+      container_name: containerName,
+      user_id: uuid,
+      seed_id: seedId,
+      seed_name: seedName, // TODO: remove when backend is implemented

```

Comment:
What is needed for this TODO to be implemented?

Reply:
currently the backend is using seed_name, it should use the id

Reply:
I'll make the change for the seed_id, but isn't the uuid useless since it's the same as the container_name? Currently, I only use the container name in the backend.

Reply:
> I'll make the change for the seed_id, but isn't the uuid useless since it's the same as the container_name? Currently, I only use the container name in the backend.

Yeah, right now they are the same thing. It's just there if we do end up having multiple containers per user.